libdnf/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ set_target_properties(libdnf PROPERTIES SOVERSION ${DNF_SO_VERSION})
 # required to have dlopen symbol
 target_link_libraries(libdnf ${CMAKE_DL_LIBS})
 # required by clang
-target_link_libraries(libdnf stdc++)
+target_link_libraries(libdnf stdc++ stdc++fs)
```

Comment:
Do you know what's the difference that this is needed on Stream but not on Fedora?

Reply:
I've added a comment explaining what stdc++fs means.
I think it's no-op on newer gcc (similar to python's future import).